### DISCUSSION TITLE
Test to compare 2 values returned from a query

### DISCUSSION THREAD

**Discussion by kevarnold972** [kevarnold972/pbi-dataqualitymonitor#10] [Ideas]
I believe that a test to compare 2 values returned in the same query would make it easier to define a test. An added value would be to compare that the values difference is within a range.

**Comment by kevarnold972** [comment]
Should this be a new test type or is this something that could be compared within the query and then use another test type for the assertion? Maybe use the #6 single value within range?

**Reply by Milhouse** [reply]
That might be the way to go. One use case is to check if Year Over Year (YoY) results are reasonable. Of course, if one already has a YoY measure, that could be the number to test. But in case of comparing model number to another model, say comparing the Sales from model to Sales from the official database number, then those would be too separate measures to compare.
